src/main/resources/application.properties

```diff
@@ -1,3 +1 @@
-server.port = 3030
-server.servlet.session.cookie.secure=true
-server.servlet.session.cookie.http-only=true
\ No newline at end of file
+userportal.url=https://testuserportal.kpmp.org
```

Comment:
Have we added anything to a release card to make sure to switch this over to the production user portal?

Reply:
https://docs.google.com/spreadsheets/d/1qohYQ6tb0KrwK6-pFo2LUBlPjZQ_cKHJN9gT79LXsnc